gravitee-apim-portal-webui-next/src/app/api/api-details/api-tab-subscriptions/subscriptions-table/subscriptions-table.component.html

```diff
@@ -16,20 +16,20 @@
 
 -->
 @if (subscriptionsList$ | async; as subscriptions) {
-  <mat-form-field appearance="outline">
-    <mat-label i18n="@@subscriptionStatusSelectLabel">Status</mat-label>
-    <mat-select [formControl]="subscriptionsStatus" multiple id="api-tab-subscription__select">
-      @for (status of subscriptionStatusesList; track status) {
-        <mat-option [value]="status">{{ status | capitalizeFirst }}</mat-option>
-      }
-    </mat-select>
-  </mat-form-field>
   @if (subscriptions.length === 0) {
     <div class="api-tab-subscription__empty" id="no-subscriptions">
       <header i18n="@@noSubscriptionAvailable" class="api-tab-subscription__empty-header">No subscription found</header>
       <p i18n="@@subscribeToApi">Subscribe to our API and your subscription will show up here.</p>
     </div>
   } @else {
+    <mat-form-field appearance="outline">
+      <mat-label i18n="@@subscriptionStatusSelectLabel">Status</mat-label>
```

Comment:
Dummy question time: why do you use the double "at" sign?

Reply:
The `@@` allows us to denote a custom ID for i18n translations. If we don't provide `@@`, an auto-generated id is created. You can read more [here](https://v17.angular.io/guide/i18n-optional-manage-marked-text).